cfgen/cfgen

```diff
@@ -406,6 +424,33 @@ def validate_profiles_desc(profiles_desc: List[Dict[str, Any]],
             f'Profile {name!r} has non-unique pool references'
 
 
+def validate_fdmi_filters_desc(fdmi_filters_desc: List[Dict[str, Any]],
+                               nodes_desc: List[Dict[str, Any]]) \
+        -> None:
+    assert all_unique(x['name'] for x in fdmi_filters_desc), \
+        'FDMI filter names are not unique:\n' + \
+        '\n'.join('  ' + x['name'] for x in fdmi_filters_desc)
+    assert all(x['name'] for x in fdmi_filters_desc), \
+        'FDMI filter name must not be empty'
+
+    Node = NamedTuple('Node', [('name', str), ('clients_nr', int)])
+    nodes = [Node(name=x['hostname'], clients_nr=x['m0_clients']['other'])
+             for x in nodes_desc]
+
+    for fdmi_filter_desc in fdmi_filters_desc:
+        node = fdmi_filter_desc['node']
+        client_index = fdmi_filter_desc['client_index']
+        assert node, f"FDMI filter {fdmi_filter_desc} " \
+            "doesn't refer to any node"
+        matching_nodes = [n for n in nodes if n.name == node]
+        assert len(matching_nodes) == 1, \
+            ("FDMI filter have exactly one node: {fdmi_filter_desc}",
```

Comment:
```suggestion
            ("FDMI filter has exactly one node: {fdmi_filter_desc}",
```

Reply:
BTW, is this message rendered correctly? I can see that the string is not formatted (`f"someting"`) so `{fdmi_filter_desc}` will not be substituted.

Reply:
It's not. You're right.

Fixed here and in other places in the patch.